daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/parsers/SequenceChildBases.scala

```diff
@@ -460,7 +460,7 @@ trait MinMaxRepeatsMixin {
 
   def erd: ElementRuntimeData
 
-  private val ock = erd.maybeOccursCountKind.get
+  final val ock = erd.maybeOccursCountKind.get
```

Comment:
The `final` was to help find the redundant definitions, yes?

Reply:
Yeah, just to make sure it's defined in only a single place so we don't accidentally override it with something that's not right.